test/gtest/uct/ib/test_cq_moderation.cc

```diff
@@ -0,0 +1,160 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2001-2017.  ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+extern "C" {
+#include <uct/api/uct.h>
+}
+#include <uct/uct_test.h>
+#include <ucs/time/time.h>
+#include <poll.h>
+#include <infiniband/verbs.h>
+
+#define TEST_PERIOD (3ul * UCS_USEC_PER_SEC) /* wait for 3 secs to get statistics */
+#define NSEC_PER_USEC (UCS_NSEC_PER_SEC / UCS_USEC_PER_SEC)
+
+#define MODERATION_PERIOD 1000 /* usecs */
+#define EVENT_LIMIT (10 * TEST_PERIOD / MODERATION_PERIOD / NSEC_PER_USEC)
+#define MAX_REPEATS 1000
+
+#define _STR(_arg) #_arg
```

Comment:
there is such macro in ucs/sys/preprocessor.h - UCS_PP_MAKE_STRING

Reply:
fixed